roles/devscripts/vars/main.yml

```diff
@@ -37,16 +37,12 @@ cifmw_devscripts_config_defaults:
   cluster_name: "ocp"
   base_domain: "openstack.lab"
   ntp_servers: "clock.corp.redhat.com"
-  ip_stack: "v4"
-  network_type: "OVNKubernetes"
```

Comment:
Removal of `network_type` causes an error on L20 in `roles/devscripts/tasks/137_custom_install.yml`.

```
The conditional check 'cifmw_devscripts_config['network_type'] == 'OVNKubernetes'' failed. The error was: error while evaluating conditional (cifmw_devscripts_config['network_type'] == 'OVNKubernetes'): 'dict object' has no attribute 'network_type'. 'dict object' has no attribute 'network_type'
```

Reply:
yep, saw it, that's why I re-added it (with a comment)